python/tvm/contrib/hexagon/pytest_plugin.py

```diff
@@ -56,13 +56,16 @@ def _compose(args, decs):
 requires_hexagon_toolchain = tvm.testing.requires_hexagon(support_required="compile-only")
 
 
-@pytest.fixture(scope="session")
-def android_serial_number() -> Optional[str]:
```

Comment:
can you explain why is it not a fixture anymore?

Reply:
Since `pytest_configure(config)` is a pytest hook, there were no easy way to have it use a pytest fixtures. 
Maybe I'm missing the point, but I did not see the usefulness of having the android_serial_number as a` pytest.fixture` and by removing it, it solved the previous error.